### DISCUSSION TITLE
Move envelope icon to end of nickname

### DISCUSSION THREAD

**Discussion by CoBC** [BearWare/TeamTalk5#1451] [Ideas]
Currently when we have an unread private message, an envelope icon is add before nickname. But, this behavior can break order of nicknames in channels tree, and make dificult to find someone using first nickname letter. I suggest to move this envelope to the end of nickname.
More, we can also add (optionally) a sound which is played when arrowing to someone with an unread private message in channels tree.
What do you think about this?

**Comment by lbk2907** [comment]
@CoBC

I agree with your suggestion.

**Comment by bear101** [comment]
Good idea!

**Comment by lbk2907** [comment]
@bear101
@CoBC
Just want to tell a minor bug here. Maybe it can be fix along with the new
feature.

1. Sometime it cut off some sound.
2. In TeamTalk version 5.10, for example, after I choose the input and
output device, I can directly press enter and it will press the ok button
and save the changes. After I update to the latest beta version of TeamTalk
that is version 5.11.0.5092 - Unreleased, when I directly press
enter when focusing on the combo box, it will not click on ok button.
Another example is in the new client instants dialog. Before this, I can
just press enter after I choose the profile to open that profile. But for
now, I need to press tab to go to ok button and press space or enter on it
to open. For the input and output device in preferences, after I change the
input or output device, I need to press tab to move the focus away from the
combo box before I can press enter to save the changes.

**Comment by alessio0292** [comment]
I can't test this thing because I have got windows seven and Bjoern did a thing that I don't like He deletes the support for windows 7 in last beta. I'd like to speak with bjoern and You collaborator if is possible in a official server uk in team talk. I go to create in this server a channel for me, collaborators and Bjoern. I'd like to speak with you or send messages chat to you.
